File: Kattis/fizzbuzz.py
a, b, c = list(map(int, input().split()))
for i in range(1, c + 1):
  if i % a == 0:
    if i % b == 0:
      print("FizzBuzz")
    else:
      print("Fizz")
  elif i % b == 0:
    if i % a == 0:
      print("FizzBuzz")
    else:
      print("Buzz")
  else:
    print(i)

File: leetcode/fizz-buzz.cs
public class Solution {
  public IList<string> FizzBuzz(int n) {
    var ret = new List<string>();
    for (int i = 1; i <= n; ++i) {
      if (i % 5 == 0 && i % 3 == 0)
        ret.Add("FizzBuzz");
      else if (i % 5 == 0)
        ret.Add("Buzz");
      else if (i % 3 == 0)
        ret.Add("Fizz");
      else
        ret.Add(i.ToString());
    }
    return ret;
  }
}
